client/src/components/organisms/LanguageSelector/LanguageSelector.component.jsx

```diff
@@ -73,7 +73,12 @@ export default function LanguageSelector() {
           variant="dark"
           id="dropdown-basic"
           onToggle={resetSearchBar}
-          style={{ textTransform: "capitalize" }}
+          style={{
+            textTransform: "capitalize",
```

Comment:
@Navdeep-av - Change this to styled component

Reply:
@xDAnkit I fixed this.